scripts/benchmarks/pytorch/run_yolox_large_multi.yml

```diff
@@ -2,5 +2,6 @@ nodes:
 - input.visual:
     source: data/benchmark/multi
 - model.yolox:
+    model_format: pytorch
```

Comment:
Is there a reason why `model_format: pytorch` is needed for YOLOX but `model_format: tensorflow` is not needed for MoveNet models?

Reply:
Added `model_format: tensorflow` to MoveNet models benchmarking scripts.
Also updated the model accordingly: it was not capturing `model_format` within `self`.